### PR TITLE
✨ Add (Un)Mute commands

### PR DESCRIPTION
**Pull request by foltone**
These are two commands that allow to mute and unmute a user with the discord feature (timeout).

prevue_mute_unmute

### CODE REVIEW

**Comment by martin-fullstacks** [comment]
There are some problems in your code... isn't `getmember` but `get user`. Please test them before send a pr.

File: slscommands/moderation/mute.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ],
+
+    run: async (client, interaction, args) => {
+        const target = interaction.options.getMember('target');
+        const duration = interaction.options.getString('duration')
+        const convertedTime = ms(duration);
+        const reason = interaction.options.getString('reason');
+
+        if (!target.moderatable) return interaction.editReply('This user cannot be muted!');
+        if (!convertedTime) return interaction.editReply('Specify a valid duration!')
```
Comment (a9ex) [inline slscommands/moderation/mute.js:45]: Max timeout time allowed by Discord is 28 days (1000 * 60 * 60 * 24 * 28)

Comment (foltone) [reply slscommands/moderation/mute.js:45]: Oh yes thanks i hadn't thought of that, I add that now.
And for the untimout it doesn't exist I searched.

**Comment by foltone** [comment]
for t "force-pushed" I simply changed the name of the commit because it didn't fit with the changes I made.

**Review by a9ex — APPROVED** [review]
Remove the unused imports, and it should be good

**Review by a9ex — APPROVED** [review]

**Comment by a9ex** [comment]
Oops I didn't see that the project was still in 13.1, so this PR won't work, I'll update it to 13.7 soon
Thanks @Mah0o !

**Pull request merged**